src/overlays/actors/ovl_En_Kakasi/z_en_kakasi.c

```diff
@@ -387,7 +387,7 @@ void EnKakasi_SetupDialogue(EnKakasi* this) {
 
 void EnKakasi_RegularDialogue(EnKakasi* this, PlayState* play) {
     u32 saveContextDay = gSaveContext.save.day;
```

Comment:
```suggestion
    u32 day = gSaveContext.save.day;
```

Reply:
`kakasi.c` was intentionally ignored for when I get to the `En_K*` cleanup, but I can clean it up here if you'd like

Reply:
Sure it can wait.